app/controllers/api/v1/users_controller.rb

```diff
@@ -29,6 +31,6 @@ def auto_login
   private
 
```

Comment:
- [ ] When you try to log in if the user is not registered please try to register the user. You have done the authentication but try to link the create method if the user not registered.
  - As we agreed we don't need `/signup` API is so we need the `/login` API to Perform create a user